### PR TITLE
Tuples

### PR DESCRIPTION
**Pull request by almargit**

### CODE REVIEW

File: learn/variables/tuple/convert.md
Comment (angarsky) [inline learn/variables/tuple/convert.md:1]: I would say that next sections are related to the `basic_operations` page:

- Tuple Inside a List in Python
- Sorted List of Tuples
- Appending a Tuple

File: learn/variables/tuple/convert.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
+Sometimes it is necessary to convert a tuple to a different data type, such as a list or a string, to perform certain operations. In some cases we need to convert some data types to tuple. Python provides built-in functions for tuple conversion, which can be used to manipulate and process data stored in tuples. Let's explore the various ways of converion and most common cases.
+
+## Tuple vs List in Python
+
+In Python, both tuples and lists are used to store collections of items, but there are some key differences between them. Here are some of the main differences:
+
+- Mutability: Lists are mutable, which means that their contents can be changed, added, or removed after they are created. Tuples, on the other hand, are immutable, which means that their contents cannot be changed once they are created.
```
Comment (angarsky) [inline learn/variables/tuple/convert.md:7]: Wrap into `**Bold**` the terms like "Mutability", "Syntax", etc.

**Review by angarsky — CHANGES_REQUESTED** [review]
I found only 2 things that should be modified in the text.

File: learn/variables/tuple/convert.md
Comment (almargit) [reply learn/variables/tuple/convert.md:1]: Transfered

File: learn/variables/tuple/convert.md (hunk 1 above)
Comment (almargit) [reply learn/variables/tuple/convert.md]: Done

**Review by angarsky — APPROVED** [review]

**Pull request merged**